tests/test_globals.hpp

```diff
@@ -31,10 +31,11 @@ struct print_log_value<glm::vec3> {
         s << glm::to_string(v);
     }
 };
+BOOST_NS_MAGIC_CLOSING
 }
 }
-BOOST_NS_MAGIC_CLOSING
 
+#if BOOST_VERSION < 106400
```

Comment:
using [wandbox.org](https://wandbox.org), I've verified that the `nullptr_t` specialization is included in boost starting from version 1.64.0.
1.63.0 needs the specialization: https://wandbox.org/permlink/XL4uPZWetPGcMbT9
1.64.0 fails to compile: https://wandbox.org/permlink/F8qO7hUFbX3C4ITE